src/components/Toggle/index.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
+import { Story, Meta } from '@storybook/react/types-6-0';
+
+import Toggle, { ToggleProps } from './';
+
+export default {
+	title: 'Toggle',
+	component: Toggle,
+	argTypes: {},
+} as Meta;
+
+const Template: Story<ToggleProps> = (args) => (
+	<div>
+		<Toggle {...args} name="radio-0">
+			Option 1
+		</Toggle>
+		<Toggle {...args} name="radio-0">
+			Option 2
+		</Toggle>
+		<Toggle {...args} name="radio-0">
+			Option 3
+		</Toggle>
+		<Toggle {...args} name="radio-0">
+			Option 4
```

Comment:
Can you please make one of the toggles without the label? Just to show it's possible

Reply:
Edited the Toggle stories to include toggles without label